Automattic/jetpack
# Backup: Move capabilities fetch functionality to hook

Organizes code a bit removing unneeded noise from the main Admin component

#### Changes proposed in this Pull Request:

* Introduces new hook
* Updates Admin to use it

#### Other information:


#### Jetpack product discussion
None

#### Does this pull request change what data or activity we track or use?
Noe

#### Testing instructions:

#### Testing instructions:

1. Launch a [JN site with Backup standalone and this branch](https://jurassic.ninja/create/?jetpack-beta&branches.jetpack-backup=update/backup-move-fetch-Capabilities-code-into-hooks&wp-debug-log)
2. Keep the JS console open and expect to see no warning for the following steps
3. Connect, expect to see the in-progress status
4. Wait until backup is complete. Expect to see information about the backup.
